live-site/lib/data.ts

```diff
@@ -1,11 +1,11 @@
 import { TabInfo } from './types';
 
 const liveSiteTabInfo: TabInfo[] = [
-  { name: 'Schedule', link: '#schedule' },
-  { name: 'Mentors', link: '#mentors' },
-  { name: 'Challenges', link: '#challenges' },
-  { name: 'Resources', link: '#resources' },
-  { name: 'Team', link: '#team' }
+  { name: 'Schedule', link: '/#schedule' },
```

Comment:
tbh not quite sure what you mean, can you attach a video?

Reply:
Yep! When I click calendar from the code of condcut page it should link to the section on the first click not the second
https://user-images.githubusercontent.com/59738880/208748263-ae0e91b5-c2d6-4ca8-b156-030f80d7e6ea.mov

Reply:
fixed this in a super hacky way - see gatsby-browser.ts. the code in that file runs immediately, but our page takes longer to load. so the code wouldn't be able to find the section of the page to scroll to, as it hadn't loaded in yet. so i added the timeout to make the code run after a second (should i make this longer to accommodate for people with really slow computers?).

Our page most likely takes longer to load because we are not using gatsby's native `Link` component  ([docs](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/)) - we are using normal anchor tags `<a> <a/>` instead. some tech debt to fix for the future. reason i didn't fix this now is that we are styling these anchor tags in the shared ui folder, and that folder does not have access to gatsby components since gatsby is only in the package.json for the main-site and live-site folders. I didn't want to import the whole library just to use one component so I left it as is.